roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -35,7 +35,7 @@
     docker_options: >
       --insecure-registry={{ openshift.docker.hosted_registry_network }}
       {{ openshift.docker.options | default ('') }}
-  when: openshift.docker.hosted_registry_insecure | default(False) | bool
+  when: openshift.docker.hosted_registry_insecure | default(False) | bool and openshift.docker.hosted_registry_network is defined
```

Comment:
Discussed on IRC but the playbook still failing was etcd, and it didn't feel right requiring that one to pass in the registry network, so started looking at handling it more gracefully internally.